src/azul/drs.py

```diff
@@ -64,8 +64,8 @@ def __str__(self) -> str:
 
 @attr.s(auto_attribs=True, kw_only=True, frozen=True)
 class Access:
-    method: AccessMethod
     url: str
+    method: AccessMethod = AccessMethod.https
```

Comment:
I don't think the default applies generally. It's entirely possible that `GET /objects/{object_id}/access/{access_id}` returns a `file:` URL.